package.json

```diff
@@ -90,7 +91,7 @@
 		"nest:start:console:debug": "nest start console --debug --watch --",
 		"nest:start:deletion-console": "nest start deletion-console --",
 		"nest:start:deletion-console:dev": "nest start deletion-console --watch --",
-		"nest:start:deletion-console:debug": "nest start deletion-console --debug --watch --",
+		"nest:start:deletion-console:debug": "nest start deletion-console --debug --watch -- execution trigger",
```

Comment:
not sure but wanted to leave my thought about it: would "execution trigger" needed to be set with quotation marks?

Reply:
actually, this should be removed as deletion console has 2 commands queue and trigger